OracleFMWInfrastructure/dockerfiles/12.2.1.3/Dockerfile

```diff
@@ -91,5 +97,6 @@ COPY --from=builder --chown=oracle:oracle /u01 /u01
 RUN chmod +xr $SCRIPT_FILE
 
 USER oracle
+HEALTHCHECK --start-period=30s --interval=30s --timeout=300s  --retries=20 CMD curl -k -s --fail `$HEALTH_SCRIPT_FILE` || exit 1
```

Comment:
Can these time values be controlled by variables?

Reply:
No these can not be controlled because the healthcheck is created at buildtime and not runtime.

Reply:
Using these values, it will take 1h20m for the container to be marked as failed. I would strongly encourage the timeout to be reduced down to something more reasonable like 5-10s and the retries also down to 3-5.